roles/config-rh-sso/tasks/create_realm.yml

```diff
@@ -1,28 +1,30 @@
 ---
 
-- name: "Retrieve template for realm: {{ realm.name }}"
+- name: "Retrieve template for realm: {{ realm_data.realm_name }}"
   set_fact:
     realm_json: "{{ lookup('template', 'realm.json.j2') | to_json }}"
 
 - name: "Requesting Access Token"
   uri:
-    url: "http://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/realms/master/protocol/openid-connect/token"
+    url: "https://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/realms/master/protocol/openid-connect/token"
```

Comment:
http v.s. https needs to be configurable otherwise this won't work if/whens someone looks to leverage just plan http.

Reply:
Added the ability to change this through the `sso_protocol` variable - suggestion from @tylerauerbeck